File: Algoriths and DataStructures/practice.py
# import random
# 
# random_integer = random.randint(1, 10)
# print(random_integer)
# 
# random_float = random.random()
# print(random_float)
# 
# states = ["Texax", "Delaware", "Pennsylvania"]

import random

namesString = input("Give me everybody's names, separated by a comma. ")
names = namesString.split(", ")

person = names[random.randint(0, len(names)-1 )]

print(f"{person} is going to buy the meal today!")

File: Algoriths and DataStructures/practice.js
// const randomIntger = Math.floor(Math.random()*10)+1
// console.log(randomIntger)
// 



const namesString = "Angela, Ben, Jenny, Michael, Chloe"
const names = namesString.split(", ")

let person = names[Math.floor(Math.random()*names.length)]
console.log(`${person} is going to buy the meal today!`)